packages/react-i18n/src/i18n.ts

```diff
@@ -431,7 +431,7 @@ export class I18n {
       : `${formattedAmount}${shortSymbol.symbol}`;
 
     return amount < 0
-      ? `-${formattedWithSymbol.replace('-', '')}`
+      ? `-${formattedWithSymbol.replace(/[-−]/, '')}`
```

Comment:
Shouldn't this use whatever negative symbol is used by the language?

(Why does this logic exist in the first place?)

Reply:
I guess for now I won't go again Polaris especially since the current Rails page format all currencies with the minus sign at the front and reverting this logic would result in React & Rails being different and could cause us problems with things being consistent while we're still webifying sections. Maybe once we're at 100% React we can revisit this decision.

For now I'll try to push through with this bug fix